tests/playwright/pages/widgets/google-maps.ts

```diff
@@ -1,17 +1,10 @@
-const EditorPage = require( '../editor-page' );
 import EditorSelectors from '../../selectors/editor-selectors';
 import Content from '../elementor-panel-tabs/content';
 
 export default class GoogleMaps extends Content {
-	constructor( page, testInfo ) {
-		super( page, testInfo );
-		this.page = page;
-		this.editorPage = new EditorPage( this.page, testInfo );
-	}
-
-	async setGoogleMapsParams( args = { location, zoom, height } ) {
+	async setGoogleMapsParams( args : { location: string, zoom: number, height: number } ) {
 		await this.page.locator( EditorSelectors.googleMaps.location ).fill( args.location );
-		await this.page.getByLabel( 'Zoom' ).fill( args.zoom );
+		await this.page.getByLabel( 'Zoom' ).fill( String( args.zoom ) );
```

Comment:
fixed. But getHeight(); returns number. I'll have to do the conversion to string in other places. Also because playwright cannot type numbers we have to add conversions in other places.